sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/FeatureManagementProperties.java

```diff
@@ -105,13 +92,57 @@ private void addToFeatures(Map<? extends String, ? extends Object> features, Str
         }
     }
 
+    private void addServerSideFeature(Map<? extends String, ? extends Object> features, String key, String combined) {
+        final Object featureValue = features.get(key);
+        if (!combined.isEmpty() && !combined.endsWith(".")) {
```

Comment:
I don't think the "combined" thing should be an issue. The combined issue is to deal with the fact that the name is part of the key in the client schema. With the client side schema the id filed will be the name.

Reply:
For server side schema, the key should be specified by `id` feild, so we don't need to deal with combined issue.

Reply:
Do we support the following properties? I add an additional "FeatureSet" field. After parse, shall we get no feature flags or get a feature flag with key"FeatureSet.Alpha"?
```
// application.yaml
FeatureManagement:
    FeatureSet:
        FeatureFlags:
            -
                id: Alpha
                enabled: true
                conditions:
                    client_filters:
                        -
                            name: Microsoft.Random
                            parameters:
                                Value: 50
```